overlay/generic/lib/svc/method/net-physical

```diff
@@ -219,6 +219,73 @@ function vnic_up
     fi
     eval "vnic_${iface}_up=true"
 }
+function vnic_up6
+{
+    set -o xtrace
+
+    typeset link="$1"
+    typeset iface="$2"
+    typeset ip="$3"
+    typeset vlan_id="$4"
+    typeset mac_addr="$5"
+    typeset mtu="$6"
+    typeset details="link='${link}', iface='${iface}', ip6='${ip}', vlan_id='${vlan_id}'"
```

Comment:
@sjorge This line, along with several others, are greater than 80 characters. It looks like there are already some long lines in this file, but I'd like to avoid adding more. Can you break the new lines up?